### ISSUE TITLE
Add <p> tag before and after when insertNode

### ISSUE THREAD

**Issue by timdown** [timdown/rangy#192]
_From [satish.rathore@gmail.com](https://code.google.com/u/satish.rathore@gmail.com/) on November 28, 2013 07:02:23_

What steps will reproduce the problem? 1. Selectt a tag Like <p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';"><font size="2" style=" font-size: 10pt;">Indicate by check mark whether the registrant is a shell company (as defined in Rule&nbsp;12b-2 of the Exchange Act).&nbsp;&nbsp;o&nbsp;Yes&nbsp;No x</font></p> <p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';">&nbsp;</p>
2.noew range.deleteContents();
       range.insertNode(newSpan);
3. After inserting Node output is look like this :

<p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';"><font size="2" style=" font-size: 10pt;"></font></p>

<p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';"><font size="2" style=" font-size: 10pt;">Indicate by check mark whether the registrant is a shell company (as defined in Rule&nbsp;12b-2 of the Exchange Act).&nbsp;&nbsp;o&nbsp;Yes&nbsp;No x</font></p> <p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';">&nbsp;</p>
<p style=" margin: 0in 0in .0001pt; text-indent: .25in; font-family: 'Times New Roman';"></p>

Basically it add a <p> tag on Start and at the End What is the expected output? What do you see instead? it should not add . What version of the product are you using? On what operating system? Version: 1.2.3 Please provide any additional information below.

_Original issue: http://code.google.com/p/rangy/issues/detail?id=192_
